osf/utils/migrations.py

```diff
@@ -215,3 +235,31 @@ def database_backwards(self, app_label, schema_editor, from_state, to_state):
 
     def describe(self):
         return 'Removes waffle flags: {}'.format(', '.join(self.flag_names))
+
+
+class AddWaffleSwitch(Operation):
```

Comment:
This should be made consistent with `AddWaffleFlags` by making it accespt a list of flag names rather than a single flag.

Reply:
I chose for the operation to take a single switch so that active can be set for each switch. It seemed better to use the operation like this: 
```
AddWaffleSwitch('some_feature', active=True)
AddWaffleSwitch('some_other_feature', active=False)
```
than like this: 
```
AddWaffleSwitch(['some_feature', 'some_other_feature'], active=True)
```
or this: 
```
AddWaffleSwitch({name: 'some_feature', active: True},  {name: 'some_other_feature', active: False})
```

I considered changing `AddWaffleFlags` to be consistent with my changes, but that seemed out of scope.

Reply:
@caseyrollins Fair enough. If we only need to add one switch at a time, we could always do 

```python
AddWaffleSwitches(['some_feature'], active=True)
AddWaffleSwitches(['some_other_feature'], active=False)
```

I think it'd be better to keep consistency now. If we decide to switch to singular (Switches, Flags), then we can do that all at once later.